index.html

```diff
@@ -57,6 +57,266 @@ <h1 id="primary-text">
                 <img src="icons/union.svg" />
             </button>
         </div>
+
+        <div id="recent-works">
+            <p class="text">My Recent Works</p>
+        
+            <div id="arrow-ctn">
+                <img src="icons/indicator.svg" />
+            </div>
+        </div>
+        
+        <section id="works">
+            <div class="work-card">
+                <div class="top-area">
+        
+                </div>
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
```

Comment:
- [ ] According to your project requirements, your technologies list must use an unordered list -  `<ul><li></li></ul>` to achieve the view.
- [ ] To avoid messing your PR conversation with repetition, kindly use this on each of your `project card`